docs/snippets/defaults_channel_mapping.mdx

```diff
@@ -27,15 +27,15 @@ default_channel_by_device:
 default_device_by_channel:
   social:   phone
   ctv-bvod: tv
-  streaming-video: tv
+  streaming-video: pc
   audio:    phone
   app:      phone
   web:      pc
 default_platform_ad_format_identifier_by_channel:
   dooh:            landscape-dooh
   social:          1080-1920-sponsored-post-social-phone
   ctv-bvod:        15s-video-ctv-bvod-tv
-  streaming-video: 15s-video-streaming-video-tv
+  streaming-video: 15s-video-streaming-video-pc
```

Comment:
@pymble2073 Can we please move this change to a follow up PR. We need to sequence this change because of a necessary update to our backend to prevent this new default taking effect immediately